src/main/java/seedu/address/model/person/Tutorial.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's tutorial in StudentConnect.
+ * Guarantees: immutable;
+ */
+public class Tutorial {
+
+    public static final String MESSAGE_CONSTRAINTS = "Tutorials should be integers between 01 and 22, "
+                                                        + "separated by a comma.";
+
+    public final String value;
+
+    /**
+     * Constructs a {@code Tutorial} with the specified tutorial value.
+     *
+     * @param tutorial The tutorial value. Must not be null.
+     */
+    public Tutorial(String tutorial) {
+        requireNonNull(tutorial);
+        checkArgument(isValidTutorial(tutorial), MESSAGE_CONSTRAINTS);
+        value = formatTutorial(tutorial);
+    }
+
+    /**
+     * Formats the tutorial to have leading zeros for numbers between 1 and 9 for consistency.
+     */
+    private String formatTutorial(String tutorial) {
```

Comment:
`VALIDATION_REGEX` forces valid tutorials to have a '0' before a single digit number, so I think this method may not be needed